chad6/template3a-tarot.js

```diff
@@ -0,0 +1,47 @@
+// This IIFE should create a subclass (constructor TarotCard)
+//  of a superclass (constructor Card)
+var TarotCard = (function(){
```

Comment:
Looks good! As an improvement, add functionality to your methods that utilizes isValid, so that if a card is not valid, is is unable to give information like rank, name, or suit. Good work.